src/figure_settings.py

```diff
@@ -54,8 +54,12 @@ def __init__(self, application, highlighted=None):
             self.props.figure_settings, self, ignorelist=ignorelist,
         )
         styles_ = styles.get_available_stylenames()
-        style_index = styles_.index(
-            self.props.figure_settings.get_custom_style())
+        try:
+            style_index = styles_.index(
+                self.props.figure_settings.get_custom_style())
+        except ValueError:
+            style_index = styles_.index("adwaita")
+            self.props.figure_settings.set_custom_style("adwaita")
```

Comment:
I am not 100% convinced that it should also change the style directly to adwaita. I just thought it may look weird that the set style does not match with Adwaita. 

On the other hand, I think it may be more preferable for people if it didn't suddenly override the style (and likely all line properties as well). So I'm thinking about removing line 62. Furthermore, in that case we can also just take index 0 (the top choice), but again adwaita seemed like a sensible default.